release/jobs/health_monitor/templates/health_monitor_ctl

```diff
@@ -27,6 +28,10 @@ case $1 in
       cp $CONFIG_DIR/uaa.pem $STORE_DIR
     fi
 
+    if [ "$HM_SYSLOG_EVENT_FORWARDER_ENABLED" == "true" ]; then
```

Comment:
inline <%= p('hm.syslog_event_forwarder_enabled') %> and remove properties.sh.erb